### DISCUSSION TITLE
What treeview solution do we use?

### DISCUSSION THREAD

**Discussion by lifeparticle** [eungella-io/Killara#33] [Q&A] [answered]
It seems like there isn't a treeview component in [mantine](https://github.com/mantinedev/mantine). Read more [here](https://github.com/mantinedev/mantine/discussions/1051).

**Comment by yeasirh** [comment, marked as answer]
Worth looking into it https://github.com/minop1205/react-dnd-treeview .

**Reply by lifeparticle** [reply]
Looks interesting!

**Reply by lifeparticle** [reply]
Let's go with [this](https://github.com/mantinedev/mantine/discussions/1051#discussioncomment-3329614).

**Comment by yeasirh** [comment]
To install, use the following command

`yarn add react-dnd @minoru/react-dnd-treeview`
